### PR TITLE
patches/schedule: add January 2021 patch schedule

### PR DESCRIPTION
**Pull request by cpanato**
#### What type of PR is this:

/kind documentation

#### What this PR does / why we need it:
patches/schedule: add January 2021 patch schedule

#### Which issue(s) this PR fixes:

None

#### Special notes for your reviewer:

/assign @justaugustus @saschagrunert @hasheddan

### CODE REVIEW

File: releases/patch-releases.md [hunk 1]
```diff
@@ -47,21 +47,31 @@ releases may also occur in between these.
...
 
+### 1.20
+
+Next patch release is **1.20.1**
+
+End of Life for **1.20** is **2021-12-30**
+
+| PATCH RELEASE | CHERRY PICK DEADLINE | TARGET DATE |
+|--- |--- |--- |
+| 1.20.1        | 2021-01-08           | 2021-01-13  |
```
Comment (saschagrunert) [inline releases/patch-releases.md:64]: I'm happy with the date and also wondering if we want to shift +1 week to ensure that most people are back from their holidays. On the other hand I do not expect many changes coming in during Christmas.

WDYT?
cc @justaugustus

Comment (cpanato) [reply releases/patch-releases.md:64]: I was using the dates that were defined, but I agree that we can push a week

Comment (hasheddan) [reply releases/patch-releases.md:64]: 1/13/21 feels like enough time to me for folks to get back from holiday, but bumping it out would be fine as well.

**Comment by cpanato** [comment]
@hasheddan @saschagrunert @tpepper should we update the dates to `1/13/21` ? or keep them as is?

**Pull request merged**